core/arch/aarchxx/mangle.c

```diff
@@ -665,6 +665,8 @@ insert_push_all_registers(dcontext_t *dcontext, clean_call_info_t *cci,
     dstack_offs += 15 * XSP_SZ;
 
     /* Make dstack_offs 8-byte algined, as we only accounted for 17 4-byte slots. */
```

Comment:
The accounting in this statement probably does not include the adjustment made to dstack_offs for proc_num_simd_registers at L600. Does it assume it must be a multiple of 8? Do we need an ASSERT?